neural/predict_with_neural.py

```diff
@@ -0,0 +1,217 @@
+import os
+import pandas as pd
+from pytorch_pretrained_bert import BertTokenizer
+import requests
+import torch
+
+import autodiscern.transformations as adt
+from neural.data_processor import DataDictProcessor
+from neural.dataset import generate_docpartition_per_question
+from neural.model import BertEmbedder, generate_sents_embeds_from_docs
+from neural.run_workflow import predict_neural_discern, get_saved_config, return_attnw_over_sents
+from neural.utilities import create_directory, ReaderWriter, get_device
+from neural.neural_discern_run_script import load_biobert_model
+from typing import Dict, List
+
+
+BASE_DIR = '/opt/data/autodiscern'
+questions = [4, 5, 9, 10, 11]
```

Comment:
Maybe writing it ALLCAPS to signify it is a global variable is helpful when spotting it within functions